### DISCUSSION TITLE
I want more programming languages as rtx cores.

### DISCUSSION THREAD

**Discussion by CHRIBUR0309** [jdx/mise#2106] [General]
Is there any plan?

**Comment by jdx** [comment]
which one do you want?

**Comment by CHRIBUR0309** [comment]
Nowadays I want to use C# (.NET?). But I would be happy to have you add programming languages one by one to the rtx core as fast as it does not interfere with your rtx development process.

**Comment by x80486** [comment]
`Erlang/OTP`, `Deno`, `Elixir`, `Elm`, `Perl`, and `Swift` are pretty good candidates to add to.

On the _package management_ side, `Maven` and `Gradle`, if applicable.

**Comment by steveno** [comment]
@jdx What are the guidelines for core modules? If someone submitted an MR to add a new plugin -  say, Ocaml, would you accept it? Are there languages/plugins you wouldn't accept?

**Comment by apps4everyone** [comment]
swift would be nice

**Comment by andersfylling** [comment]
I suggest moving this to discussions and create a community poll to avoid clutter: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll

**Comment by CHRIBUR0309** [comment]
> I suggest moving this to discussions and create a community poll to avoid clutter: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll

Good idea.
